adapters/zeroclickfraud/usersync.go

```diff
@@ -0,0 +1,12 @@
+package zeroclickfraud
+
+import (
+	"text/template"
+
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/usersync"
+)
+
+func NewZeroclickfraudSyncer(temp *template.Template) usersync.Usersyncer {
```

Comment:
Is the proper capitalization of your `Zeroclickfraud` or `ZeroClickFraud`? If it's the latter, please consider updating for consistency.

Reply:
Thanks, I missed that. Will fix for consistency.